concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,74 @@
+from django.core.checks     import Error, register
+from django.conf            import settings
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
+
+
+@register()
+def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+
+    concent_features = settings.CONCENT_FEATURES
+    if not isinstance(concent_features, list):
+        errors.append(
+            Error('CONCENT_FEATURES error:', hint = 'CONCENT_FEATURES should be a list.', id = 'settings_check_001')
+        )
+        return errors
+
+    check_for_duplicate = []
+    for application in concent_features:
+        if application not in ["concent-api", "gatekeeper", "admin-panel"]:
+            errors.append(
+                Error('CONCENT_FEATURES error:', hint = '{} app in CONCENT_FEATURES is not allowed'.format(application), id = 'settings_check_002')
+            )
+            return errors
+
+        if application in check_for_duplicate:
+            errors.append(
+                Error('CONCENT_FEATURES error:', hint = 'Duplicate {} in CONCENT_FEATURES'.format(application), id = 'settings_check_003')
+            )
+            return errors
+        check_for_duplicate.append(application)
+
+        if application == "concent-api":
+            if "core" not in settings.INSTALLED_APPS:
```

Comment:
If it is the final version of those ifs, you can combine them into one, same for the next two.

Reply:
Done